tools/static-assets/skel-vue/package.json

```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "@babel/runtime": "^7.23.5",
     "meteor-node-stubs": "^1.2.12",
+    "meteor-vite": "^3.8.0",
     "vue": "^3.3.9",
+    "vite": "^6.3.5",
```

Comment:
It should be a dev dependency, no?
Same for "meteor-vite"

Reply:
No, it should be a production dependency as it reads and exposes assets within the client manifest through Meteor's built-in web server.